apps/etterlatte-saksbehandling-ui/client/src/components/person/sakOgBehandling/OmgjoerAvslagModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { Alert, BodyShort, Button, Heading, Modal } from '@navikt/ds-react'
+import React, { useState } from 'react'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { opprettOmgjoeringFoerstegangsbehandling } from '~shared/api/revurdering'
+import { isPending, isSuccess, mapResult } from '~shared/api/apiUtils'
+import { ApiErrorAlert } from '~ErrorBoundary'
+import { useNavigate } from 'react-router-dom'
+
+export function OmgjoerAvslagModal(props: { sakId: number; harAapenBehandling: boolean }) {
+  const { sakId, harAapenBehandling } = props
+  const [open, setOpen] = useState(false)
+  const [opprettOmgjoeringStatus, opprettOmgjoering] = useApiCall(opprettOmgjoeringFoerstegangsbehandling)
+  const navigate = useNavigate()
+  function opprett() {
+    opprettOmgjoering({ sakId })
+  }
+
+  function lukkModal() {
+    setOpen(false)
+  }
+
+  return (
+    <>
+      <Button variant="secondary" size="medium" onClick={() => setOpen(true)}>
+        Omgjør førstegangsbehandling
+      </Button>
+      <Modal open={open} onClose={lukkModal} aria-labelledby="omgjoer-foerstegangsbehandling-modal-header">
+        <Modal.Header closeButton>
+          <Heading id="omgjoer-foerstegangsbehandling-modal-header" level="2" size="medium">
+            Omgjør førstegangsbehandling
+          </Heading>
+        </Modal.Header>
+        <Modal.Body>
+          <BodyShort>
+            Hvis det kun er avslåtte / avbrutte førstegangsbehandlinger i saken må en eventuell omgjøring gjøres som en
+            ny førstegangsbehandling.
+          </BodyShort>
+          {harAapenBehandling && (
+            <Alert variant="warning">
+              Saken har allerede en åpen behandling. Denne må avsluttes eller avbrytes før en omgjøring kan startes.
```

Comment:
Vil denne noen gang slå ut siden tilsvarende sjekk brukes for å skjule modal?

Reply:
Det har oppstått i produksjon -- det er en sak som har fått feilaktige opprettede revurderinger på grunn av ny søknad (logikken der er fikset). SIden det vil feile når dette har oppstått / potensielt oppstår i framtiden la jeg på en sjekk :)